app/controllers/api/base_controller/parameters.rb

```diff
@@ -1,6 +1,8 @@
 module Api
   class BaseController
     module Parameters
+      include CompressedIds
```

Comment:
Agreed, let's do that in a separate PR, removing all other includes of CompressedIds too.